v22.2/error-handling-and-troubleshooting.md

```diff
@@ -97,30 +97,49 @@ To mitigate against this situation while keeping SCRAM authentication enabled, C
 
 {% include_cached {{page.version.version}}/scram-authentication-recommendations.md %}
 
-#### Downgrade from SCRAM authentication
+If the above steps don't work, you can try lowering the default hashing cost and reapplying the password as described below.
+
+##### Lower default hashing cost and reapply the password
+
+To decrease the CPU usage of SCRAM password hashing while keeping SCRAM enabled:
 
-As an alternative to the [mitigation steps listed above](#mitigation-steps-while-keeping-scram-enabled), you can downgrade from SCRAM authentication by taking the following steps.
+1. Set the [`server.user_login.password_hashes.default_cost.scram_sha_256` cluster setting](cluster-settings.html#setting-server-user-login-password-hashes-default-cost-scram-sha-256) to `4096`:
 
-1. Turn off the `server.user_login.cert_password_method.auto_scram_promotion.enabled` [cluster setting](cluster-settings.html#setting-server-user-login-cert-password-method-auto-scram-promotion-enabled):
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING server.user_login.password_hashes.default_cost.scram_sha_256 = 4096;
+~~~
 
-    {% include_cached copy-clipboard.html %}
-    ~~~ sql
-    SET CLUSTER SETTING server.user_login.upgrade_bcrypt_stored_passwords_to_scram.enabled = false;
-    ~~~
+2. Update all users to use strong, complex passwords using the [`ALTER USER`](alter-user.html) statement:
```

Comment:
i'd reorg this a bit - rather than making it a "step" for how to lower the default cost, can we make it a tip/suggestion. something like:

> Tip: When lowering the default cost, we recommend that you use strong, complex passwords for SQL users.